pkg/sql/catalog/lease/lease.go

```diff
@@ -1047,6 +1047,7 @@ func (m *Manager) SetDraining(
 		for _, l := range leases {
 			releaseLease(ctx, l, m)
 		}
+		delete(m.mu.descriptors, id)
```

Comment:
Just out of curiosity, what was the impact of this line being missing until now?

Reply:
I don't think anything. It just seemed bad to have released leases sitting there. This is during draining anyway. I think we would have found the released leases and then re-acquired.